### ISSUE TITLE
Problem: Blind Polling until the date matches an expression

### ISSUE THREAD

**Issue by maennchen** [c-rack/quantum-elixir#173]
Currently we blindly increase the time by one second in a loop. Then we check every job against the current date.

It could be more reasonable to calculate the next run date on every job and sleep until then. This could be achieved using [`Crontab.Scheduler.get_next_run_dates`](https://hexdocs.pm/crontab/Crontab.Scheduler.html#get_next_run_dates/2)

I think this would make the code a lot more understandable.

**Comment by c-rack** [comment]
Duplicate of #4, but would be happy to see this implemented!

**Comment by maennchen** [comment]
Great, I will do after #152

**Comment by michalmuskala** [comment]
This should be able to use [`Process.send_after`](https://hexdocs.pm/elixir/Process.html#send_after/4) with the `:abs` option

**Comment by maennchen** [comment]
Solved with #230
